src/_pytest/terminal.py

```diff
@@ -879,16 +881,13 @@ def mkrel(nodeid):
             if self.verbosity >= 2 and nodeid.split("::")[0] != fspath.replace(
                 "\\", nodes.SEP
             ):
-                res += " <- " + bestrelpath(self.startpath, fspath)
+                res += " <- " + bestrelpath(self.startpath, Path(fspath))
         else:
             res = "[location]"
         return res + " "
 
-    def _getfailureheadline(self, rep):
-        head_line = rep.head_line
-        if head_line:
-            return head_line
-        return "test session"  # XXX?
```

Comment:
We should keep this XXX comment - I don't know what it refers to but it would probably help *someone* looking at this code.